exo-deploy/src/deploy.ls

```diff
@@ -3,6 +3,10 @@ require! {
   'require-yaml'
 }
 
+command-flag = process.argv[2]
```

Comment:
Ha! Can you please document via a comment in the top of this file that this file runs inside Docker?